app/models/company.rb

```diff
@@ -31,8 +31,6 @@ class Company < ApplicationRecord
 
   enum company_type: COMPANY_TYPES
 
-  before_save { email.downcase! }
```

Comment:
do you really need to remove this?

Reply:
if you handle this in other place, it is OK

Reply:
seems weird doesn't look like you changes make sure you dont have some problems with rebasing